src/formElementPlugins/dropdownChoice/src/ApplicationView.tsx

```diff
@@ -116,16 +115,22 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
       <Markdown text={description} />
       <Dropdown
         fluid
-        //multiple
-        // onAddItem={additionalItem}
+        // multiple
         selection
         clearable
         search={search || hasOther}
         allowAdditions
-        onAddItem={(e, data)=>addItemHandler(e, data.value as string)}
+        additionLabel={strings.ADD_CUSTOM_OPTION_LABEL}
+        onAddItem={(e, data) => addItemHandler(data.value as string)}
         placeholder={placeholder}
-        options={dropDownOptions}
-        onChange={handleChange}
+        options={getDropdownOptions(
```

Comment:
Making `getDropdownOptions` a separate function ensures it'll always reflect the most up-to-date state. I was having problems with it being updated when options was slow to load.